src/npcmove.cpp

```diff
@@ -2570,11 +2570,14 @@ void npc::worker_downtime()
         move_pause();
         return;
     }
-    //  already know of a chair, go there
-    if( chair_pos != tripoint_min ) {
-        if( here.has_flag_furn( ter_furn_flag::TFLAG_CAN_SIT, here.getlocal( chair_pos ) ) ) {
-            update_path( here.getlocal( chair_pos ) );
-            if( pos() == here.getlocal( chair_pos ) || path.empty() ) {
+    //  already know of a chair, go there - if there isn't already another creature there.
+    //  this is a bit of behind the scene omniscience for the npc, since ideally the npc 
+    //  should walk to the chair and then change their destination due to the seat being taken.
+    auto local_chair_pos = here.getlocal( chair_pos );
```

Comment:
```suggestion
    tripoint local_chair_pos = here.getlocal( chair_pos );
```
You will also need to run astyle (with the settings in the main directory `.astylerc` file).

Reply:
> 
> 
> You will also need to run astyle (with the settings in the main directory `.astylerc` file).

Ah, yeah thanks.